docs/site/Repositories.md

```diff
@@ -493,7 +493,7 @@ retrieve all the rows that match a particular filter for a model instance.
 ```ts
 public find(
   modelClass: Class<Entity>,
-  filter: Filter,
+  filter: Filter<Entity>,
```

Comment:
wouldn't this be `Account`? I feel the type for `modelClass` and returned data should also be `Account` instead of `Entity`.

Reply:
I was just fixing `Filter`. But `Account` makes sense.